easybuild/easyconfigs/j/json-c/json-c-0.15-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+
+name = 'json-c'
+version = '0.15'
+local_suff = '-20200726'
+
+homepage = 'https://github.com/json-c/json-c'
+description = """JSON-C implements a reference counting object model that allows you to easily construct JSON objects
+ in C, output them as JSON formatted strings and parse JSON formatted strings back into the C representation of JSON
+objects."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/json-c/json-c/archive/']
+sources = ['json-c-%%(version)s%s.tar.gz' % local_suff]
+checksums = ['4ba9a090a42cf1e12b84c64e4464bb6fb893666841d5843cc5bef90774028882']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.21.1'),
+]
+
```

Comment:
```suggestion

# disable using Valgrind during the tests to avoid failures caused by using an OS Valgrind
pretestopts = 'USE_VALGRIND=0 '
```

This fixes https://github.com/easybuilders/easybuild-easyconfigs/pull/13748#issuecomment-899012033 where an OS installed Valgrind is used during the tests and causes the failures.

And #13750 for fixing this in the existing EasyConfigs.